test/testautomation_audio.c

```diff
@@ -163,9 +194,10 @@ static int audio_initOpenCloseQuitAudio(void *arg)
 
         /* Change specs */
         for (j = 0; j < 2; j++) {
+            SDLTest_AssertCheck(!SDL_WasInit(SDL_INIT_AUDIO), "Validate audio subsystem not initiaized");
 
             /* Call Init */
-            SDL_SetHint("SDL_AUDIO_DRIVER", audioDriver);
+            SDL_SetHint(SDL_HINT_AUDIO_DRIVER, audioDriver);
             result = SDL_InitSubSystem(SDL_INIT_AUDIO);
```

Comment:
I noticed the (new) driver hint was not applied when the audio system is still initialized.
That's why the new `audioSetUp` makes sure the audio initialiation refcount is `1`.